doc/syscall-interception.md

```diff
@@ -37,6 +37,52 @@ at all.
 
 This can be enabled by setting `security.syscalls.intercept.mknod` to `true`.
 
+### bpf
+The `bpf` system call is used to manage eBPF programs in the kernel.
+Those can be attached to a variety of kernel subsystems.
+
+In general, loading of untrusted eBPF program can be problematic as it
+can facilitate timing based attacks.
+
+LXD's eBPF support is currently restricted to allowing programs use to
```

Comment:
Something not quite right with this line:

> LXD's eBPF support is currently restricted to allowing programs use to manage devices cgroup entries.

It doesn't read correctly. Perhaps just drop the word "use"?